frontend/src/App.jsx

```diff
@@ -35,7 +35,7 @@ function App() {
         <NotificationContainer />
         <TypesContext.Provider value={types}>
           <Routes>
-            <Route path="/" element={<Bookshelf {...{backend, user}} />} />
+            <Route path="/" element={<Bookshelf createOn={[]} {...{backend, user}} />} />
```

Comment:
`createOn` is just an empty list. And `Bookshelf` is used nowhere else. So there is no reason to pass it as a parameter.